maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/pom.xml

```diff
@@ -48,8 +48,8 @@
       <artifactId>maven-scm-provider-git-commons</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-interactivity-api</artifactId>
+      <groupId>org.codehaus.plexus</groupId>
```

Comment:
Retain order please

Reply:
Done in https://github.com/apache/maven-scm/pull/155/commits/9f86604ea544ad802dffcd48e5c381facebb9269.